extensions/wikia/GlobalNavigation/scripts/GoogleSearchAPI.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * We AB test for wikia search (both within and all wikias) in Oasis with Google custom search result.
+ * window.wgGoogleSearchTest: a global variable set true in Optimizely for 50% of users who visit ja wikia.
+ * window.wgGoogleSearchParam: Search ID set in Optimizely.
+ */
+$(function () {
+	if ( window.wgGoogleSearchTest ) {
```

Comment:
Hi, This variable (window.wgGoogleSearchTest) is defined in Optimizely. I don't think I can use require for it?